assets/js/components/Dashboard/PendingScreensPage.tsx

```diff
@@ -165,7 +168,21 @@ const PendingScreensPage: ComponentType = () => {
 
   return (
     <div className="pending-screens-page">
-      <div className="page-content__header">Pending</div>
+      <div className="page-content__header">
+        <Container fluid>
+          <Row className="align-items-center text-white">
+            <Col>Pending</Col>
+            <Col className="d-flex justify-content-end">
```

Comment:
Assuming the class name `d-flex` applies `display: flex`, does this do anything in this context? The element only has one child (the button), so there's not really anything for flexbox to lay out.

Reply:
It looks like it's using `justify-content` to push the button to the right, so it needs to be a flex container for that to work.

Reply:
Ah, right, I was thinking the column layout was already handling that. Do we have a `justify-self-end` utility? Since this column is already a child of a flex layout (or grid layout? unclear to me what `fluid` actually does).

Reply:
`fluid` is something that React bootstrap has that allows you to force a `Container` to use all available horizontal space. Docs are [here](https://react-bootstrap.github.io/docs/layout/grid/#container-1). Without it, it pushes its children towards the center.

Unfortunately, Bootstrap does not have a `justify-self-*` utility, but I think I found a better way to accomplish the styling here. You're right that the `Row` is already flex by default. I can add `justify-content-between` to it and change the responsive breakpoints of the `Col`s to `auto` so they only take up as much space as they need. Then the space will be added between the elements and no `Col` elements need to be flex unnecessarily.